src/server/middleware/errorHandler.js

```diff
@@ -14,9 +14,8 @@ const errorHandler = (err, req, res, next) => {
   logEmitter.emit("error", `statusCode: ${res ? res.statusCode : err ? err.statusCode : null}`);
   var props = {
     statusCode: res ? res.statusCode : err ? err.statusCode : "500",
-    err: err ? err : "An error occurred.",
-    ...(err.stack && err.stack.toString().includes("propsGenerator") ? {} : PropsGenerator(req))
```

Comment:
This probably shouldn't be removed?  Was this just for debugging?

Reply:
I think I suggested as shouldn't be passing stack trace to views

Reply:
I guess there's probably no props from PropsGenerator required for the error page anyway